src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -113,6 +113,10 @@ class ParsedMRI:
     name: str
     unit: str
 
+    @property
+    def mri_string(self) -> str:
+        return f"{self.entity}:{self.namespace}/{self.name}@{self.unit}"
```

Comment:
also i don't see unit-test for this line.

Reply:
🤷 doesn't feel necessary to me since its a string format & already implicitly tested by is_custom_measurement. But I added an assert to the parse_mri test anyways

Reply:
agreed.